### PR TITLE
feat: add node test definition

### PR DESCRIPTION
**Pull request by jbeveland27**

### CODE REVIEW

**Review by Julien4218 — APPROVED** [review]
Looks good to me.
You could also add a little cronjob to generate a bit of traffic, see https://github.com/newrelic/demo-catalog/blob/main/catalog/hello/hello.aws.json#L39-L47

**Comment by jbeveland27** [comment]
@Julien4218 Good call - updated to add the cronjob

File: test/manual/definitions/apm/node/node-linux2.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      "destinations": [
+        "nodelinux2"
+      ]
+    },
+    {
+      "destination_filepath": "engine/cronjob.json",
+      "content": [
+        {
+          "frequency": "* * * * *",
+          "job": "for ((i=1;i<=30;i++)); do curl '[service:app1:url]/'; curl '[service:app1:url]/api/inventory'; sleep 1; done"
```
Comment (Julien4218) [inline test/manual/definitions/apm/node/node-linux2.json:32]: there is no `app1` in this config, so the field should be referenced using `nodetron` for the service_id [service:nodetron:url]

Comment (jbeveland27) [reply test/manual/definitions/apm/node/node-linux2.json]: Good catch. Fixed.

**Review by Julien4218 — APPROVED** [review]

**Pull request merged**